core-ui/src/components/Lambdas/LambdaDetails/LambdaDetails.js

```diff
@@ -6,7 +6,7 @@ import LambdaDetailsHeader from './LambdaDetailsHeader/LambdaDetailsHeader';
 import CodeTab from './Tabs/Code/CodeTab';
 import ResourceManagementTab from './Tabs/ResourceManagement/ResourceManagementTab';
 
-import EventTriggersWrapper from './Tabs/Configuration/EventTriggers/EventTriggersWrapper';
+import EventTriggersWrapper from './Tabs/Configuration/EventTriggers/LambdaEventsWrapper';
```

Comment:
Shouldn't that be
```suggestion
import LambdaEventsWrapper from './Tabs/Configuration/EventTriggers/LambdaEventsWrapper';
```
?